src/components/Icon.jsx

```diff
@@ -1,42 +1,96 @@
 import React from "react";
-import { Icon } from "@iconify/react";
-import editIcon from "@iconify/icons-fa/edit";
-import deleteIcon from "@iconify/icons-fa/trash";
-import radioButtonOff from "@iconify/icons-ion/radio-button-off";
-import radioButtonOn from "@iconify/icons-ion/radio-button-on";
-import plusLight from "@iconify-icons/ph/plus-light";
-import funnelOutline from "@iconify/icons-ion/funnel-outline";
 
-const EditIcon = () => <Icon icon={editIcon} className="cursor-pointer" />;
-
-const DeleteIcon = () => (
-  <Icon icon={deleteIcon} className="text-red-600 cursor-pointer" />
-);
+const EditIcon = () => {

```

Comment:
You are only expected to return the svg, not the button. So that later you can do

```
<button onClick={clicked}>
    <EditIcon />
</button>
```

Applies everywhere else.

Reply:
i have removed all the buttons that was used to wrap all the icons in icon.jsx

Reply:
Include `{props}` in your argument and spread them in the svg component